crd/client.go

```diff
@@ -169,7 +169,7 @@ func waitForCRDs(crdClient *rest.RESTClient) error {
 			return nil
 		}
 
-		if time.Since(start) > 30*time.Second {
+		if time.Since(start) > 60*time.Second {
```

Comment:
Why increase this?  It feels like 30 sec should be enough?

Reply:
ok, I will rollback to 30 sec. And increase it in future if it's not enough.